gaffer-as-a-service/gaas-rest/src/test/java/uk/gov/gchq/gaffer/gaas/util/GaaSGraphConfigsLoaderTest.java

```diff
@@ -89,6 +89,12 @@ public void listConfigSpecs_shouldReturnProxiesConfigSpec_whenStorePropStoreClas
         gafferSpecFederatedStore.putNestedObject("uk.gov.gchq.gaffer.sketches.serialisation.json.SketchesJsonModules", GAFFER_SERIALISER_JSON_NESTED_KEYS);
         expected.put("federated", gafferSpecFederatedStore);
 
+        final GafferSpec gafferSpecFederatedOpDeclarationStore = new GafferSpec();
```

Comment:
This doesn't match the `federatedOpDeclaration.yaml` as there are no `operationDeclarations` in this object. I think this isn't picked up in tests, however, because you are only testing the keys match, not the values too (need to use `entrySet`): https://github.com/gchq/gaffer-experimental/blob/99fc7cb396647798eb2fa2c968eb88cf22daca5e/gaffer-as-a-service/gaas-rest/src/test/java/uk/gov/gchq/gaffer/gaas/util/GaaSGraphConfigsLoaderTest.java#L113

Reply:
Nice spot updated thanks